app/javascript/packs/component-definitions-common.js

```diff
@@ -41,13 +43,15 @@ ManageIQ.component.addReact('CloudNetworkForm', CloudNetworkForm);
 ManageIQ.component.addReact('CloudTenantForm', CloudTenantForm);
 ManageIQ.component.addReact('CopyCatalogForm', CopyCatalogForm);
 ManageIQ.component.addReact('CopyDashboardForm', CopyDashboardForm);
+ManageIQ.component.addReact('DataDrivenTable', DataDrivenTable);
```

Comment:
We don't need to register this component if its not directly used in haml with `= react`.

Reply:
Ah, true, removing, and removing the `window[transform]` support for loading global functions, as that was only useful when using `DataDrivenTable` via `=react`